doc/codi.txt

```diff
@@ -65,6 +67,10 @@ displays the results of evaluating each line as you type (with NeoVim or Vim
 with |+job| and |+channel|, asynchronously). It's extensible to nearly any
 language that provides a REPL (interactive interpreter)!
 
+If you're using NeoVim, by default it uses virtual text to print the output
```

Comment:
I was following the approach other plugin devs had implement that is to make defaults which take advantage on nvim features.

Perhaps people checking out this plugin want to know right away the this feature is available in nvim, otherwise manually disable it.

In any case, we could warn users in the readme.md with this. But I strongly think this should be the default and even Vim should implement this as well, but time will tell.

Reply:
My biggest concern is that pushing this as a default feature will involuntarily change the behavior of all current NeoVim users when they update the plugin. I would be happy if we prominently showcased this feature in `README.md` (e.g. with another GIF), but I'm not comfortable with changing the default behavior.

Reply:
I changed my mind, I don't mind so much :)